client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -74,16 +77,6 @@ const onboarding: Flow = {
 					if ( providedDependencies.goToCheckout ) {
 						const siteSlug = providedDependencies.siteSlug as string;
 
-						if ( planCartItem && siteSlug && flowName ) {
-							await addPlanToCart( siteSlug, flowName, true, '', planCartItem );
-						}
-
-						if ( domainCartItem && siteSlug && flowName ) {
-							await addProductsToCart( siteSlug, flowName, [ domainCartItem ] );
-						}
-
-						resetOnboardStore();
```

Comment:
We can remove the `resetOnboardStore` variable since it's no longer used in this file.